modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -87,13 +95,20 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
     mceTableInsertColAfter: () => actOnSelection(actions.insertColumnsAfter),
     mceTableDeleteCol: () => actOnSelection(actions.deleteColumn),
     mceTableDeleteRow: () => actOnSelection(actions.deleteRow),
+    mceTableCutCol: (_grid) => copyColSelection().each((selection) => {
+      clipboard.setColumns(selection);
```

Comment:
A thought I just had... when cutting/copying a row/column should we clear the opposite clipboard data? eg if I cut a row, then cut a column, should I still be able to paste the row?

Reply:
Considering that goes against usual clipboard behaviour, I'd say it should clear